tests/test_merge.py

```diff
@@ -40,10 +41,10 @@ def test_merge_from_fixtures(tmp_path):
     and validates against expected networks,
     i.e. subnets are merged into the root network appropriately.
     '''
-    testdir = Path(__file__).parent
-    base_nets, base_results = __read_test_vectors(testdir / "data/base_file.csv")
+    data_dir = Path(__file__).parent / "data"
+    base_nets, base_subnet_count = __read_test_vectors(data_dir / "base_file.csv")
```

Comment:
There are still the old variable names here. Probably a silent merge conflict.

Reply:
sorry :disappointed:, fixed.